--- plasma-mediacenter/shells/newshell/mainwindow.cpp.orig	2015-05-21 09:48:52.584791815 +0100
+++ plasma-mediacenter/shells/newshell/mainwindow.cpp	2015-05-21 09:49:00.399792314 +0100
@@ -221,7 +221,7 @@
     typedef void (*MyPrototype)();
     typedef QString (*QStringPrototype)();
 
-    QLibrary plasmaAdapterLib("plasmaadapter.so");
+    QLibrary plasmaAdapterLib("plasmaadapter.so.1.3");
     MyPrototype loadPlasmaTheme = (MyPrototype) plasmaAdapterLib.resolve("loadPlasmaTheme");
     QStringPrototype getMainscriptPath = (QStringPrototype) plasmaAdapterLib.resolve("getMainscriptPath");
     QStringPrototype getImagesPath = (QStringPrototype) plasmaAdapterLib.resolve("getImagesPath");
